runners/wikiConsole/templates/installDevonfwIde.asciidoc

```diff
@@ -1,6 +1,7 @@
 devonfw setup
 =============
 
+If you already installed the devonfw-ide, you can skip this step.
```

Comment:
In the rest of the text the term 'devonfw IDE' is used. This should not be mixed.

Reply:
I think in most tutorials we used devonfw IDE. WE should keep it consistent to the other tutorials, too.